pyproject.toml

```diff
@@ -37,4 +37,7 @@ pandas = "1.4.*"
 scikit-learn = "1.2.*"
 uvicorn = "0.22.*"
 
-# poetry export --without-hashes --format=requirements.txt --with="style,test,webserver" --output=requirements.txt
+[tool.poetry.group.notebooks.dependencies]
```

Comment:
trzeba dodać tą grupę do makefile'a do polecenia make install

Reply:
Mam taki problem z make install.
Jedna z bibliotek, której używamy ma nową wersję — to ona się teraz ściąga. Nie wiem czy to znowu problem mojego komputera, ale nie ma teraz supportu dla makefile. I teoretycznie chyba można by ustawić na sztywno wersję, ale to jest nieładne. Druga opcja -> te jedną bibliotekę ściągać "ręcznie" (`pip install greenlet==3.0.2`) i może kiedyś naprawią.

Reply:
Co to znaczy, że nie supportuje makefile'a? Te biblioteki wiedzą, kiedy się je instaluje poleceniem "poetry install" bezpośrednio z terminala, a kiedy "poetry install" wywołanym przez polecenie make?

Reply:
Zgadzam się poetry to problem. (Nie umiem się klarownie wysłowić)